src/platform_linux/shmem.c

```diff
@@ -218,20 +218,6 @@ platform_heap_id_to_shmaddr(platform_heap_id hid)
    return shmaddr;
 }
 
-/* Evaluates to valid 'low' address within shared segment. */
-static inline void *
-platform_shm_lop(platform_heap_id hid)
-{
-   return (void *)platform_heap_id_to_handle(hid);
-}
-
-/* Evaluates to valid 'high' address within shared segment. */
-static inline void *
-platform_shm_hip(platform_heap_id hid)
-{
-   return (((shmem_info *)platform_heap_id_to_shmaddr(hid))->shm_end - 1);
-}
-
```

Comment:
I suppose you are deleting these functions as they are defined as `static` but are not used anywhere in this file.

I thought I had used it for some assertion checks / diagnostics, but can't find those usages now.

I have found these to be useful during development and would like to retain them, just so that they are available next time I have to add some diagnostic xchecks to verify that an address is within a shared segment region.

What if you just make them non-`static` and let them be in this file. Or, even remove the `inline` qualifier, if needed. Will that allow compilation to go through? If so, can we retain these two.

Reply:
Ok, I removed the `static` and that was enough to silence the compiler errors